terraform/050-pw-manager/vars.tf

```
@@ -84,23 +84,23 @@ variable "cloudwatch_log_group_name" {
 
 variable "code_length" {
   description = "Number of digits in reset code."
-  type        = string
-  default     = "6"
+  type        = number
+  default     = 6
 }
 
 variable "cpu" {
-  type        = string
+  type        = number
   description = "Amount of CPU to allocate to container, recommend '250' for production"
-  default     = "64"
+  default     = 64
```

Should default be recommendation?
```suggestion
  default     = 250
```